llvm/include/llvm/BitstreamCAS/BitstreamCASNode.h

```diff
@@ -0,0 +1,15 @@
+#include "llvm/CAS/CASID.h"
+#include "llvm/CAS/CASReference.h"
+#include "llvm/CAS/ObjectStore.h"
+
+namespace llvm::cas {
+
+struct BitstreamCASNode {
+  SmallVectorImpl<char> &Data;
```

Comment:
I feel like this should either be a SmallVector owned by the Node as scratch space for the current active stack frame to write, or it is not needed if we only write to one node at a time so we can write to the scratch space in CASWriter directly.

I suspect probably 1 is the case here.

Reply:
Yeah, it's updated as discussed with SmallVector.